vixos/__main__.py

```diff
@@ -74,7 +74,11 @@ def make_nix_config_file(self, executable: str) -> str:
         managed_config = self.vixos_path / "managed.nix"
         managed_config.write_text(generate_nix_managed(self.name, self.ssh.pubkey_text))
 
-        localf = self.vixos_root / "local.nix"
+        globalf = self.vixos_root / "global.nix"
+        if not globalf.exists():
+            globalf.write_text(generate_global_nix())
+
+        localf = self.vixos_path / "local.nix"
```

Comment:
Templates look swapped? `global.nix` should get the current template (with keyboard configuration etc).

Reply:
Bruh, both are empty.

Reply:
This generates a file that's supposed to be a default configuration for all VMs:

```
def generate_local_nix() -> str:
    return """{
  # services.xserver.xkbOptions = "ctrl:nocaps";
  # services.xserver.layout = "pl(intl)";
}"""
```

Since we're making naming changes, this should be renamed to `generate_global_nix` and go to `global.nix` file in the `vixos_root`, right?

Reply:
Just to be clear, this is what I mean:

```
msm@transient ~/vixos
❯ cat global.nix
{}                                                                                                                                                                                                                  
msm@transient ~/vixos
❯ cat bash/local.nix
{
  # services.xserver.xkbOptions = "ctrl:nocaps";
  # services.xserver.layout = "pl(intl)";
}                                                                                                                                                                                                                   
```

(should be the other way around)